code/src/components/Loader.js

```diff
@@ -0,0 +1,23 @@
+import styled, { keyframes } from "styled-components";
+
+const spin = keyframes`
+{
+  0% {
+    transform: rotate(0deg);
+  }
+  50% {
+    transform: rotate(180deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }}
+`;
+
+export const Loader = styled.div`
+  border: 16px solid whitesmoke;
+  border-top: 16px solid ${(props) => props.color};
+  border-radius: 50%;
+  width: ${(props) => props.width}px;
+  height: ${(props) => props.height}px;
+  animation: ${spin} 1s linear infinite;
+`;
```

Comment:
Very nice component, would say that it would be nice to actually see it longer than 1s so maybe 2-3 seconds. 
Also it seems that the loader is not viewable when you have scrolled down while you are choosing a movie further down in the list. You only see the loader if you stay at the top of the page. I dont know if it is a placement in the styling perhaps.